scripts/darwin/build_fast.sh

```diff
@@ -3,19 +3,48 @@
 # Load system env only
 source /etc/bashrc
 source /etc/profile
-source /projects/parthenon-int/parthenon-project/.bashrc
+source /projects/parthenon-int/parthenon-project/spack/share/spack/setup-env.sh
```

Comment:
> What's the reason for this change?

That's a good question. It is generally not necessary to call spack directly when running with the bashrc file because CMake imports all the dependencies from the machine config file. However, the python application also has dependencies these are not automatically found. What I could try to do is change the logic in the bashrc file if needed, so the application dependencies are loaded there. That way all the build dependencies are handled in the same file.